src/runtime/offers-flow.js

```
@@ -91,6 +89,8 @@ export class OffersFlow {
       if (skuId) {
         return new PayStartFlow(this.deps_, skuId).start();
       }
+    }).catch(() => {
+      this.dialogManager_.completeView(this.activityIframeView_);
```

Can't swallow the error. Let's rethrow.

---

Done